app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,6 @@
+# Controller to handle mediations for admins
+class AdminController < ApplicationController
+  def index
+    @requests = Request.all.group_by(&:origin).sort
```

Comment:
You'll probably want to use [#order](http://api.rubyonrails.org/classes/ActiveRecord/QueryMethods.html#method-i-order) for sorting active record queries, which will play nicer with Kaminari.